test/e2e/backup_script_test.go

```diff
@@ -0,0 +1,59 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"github.com/openshift/cluster-etcd-operator/test/e2e/framework"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+const (
+	masterNodeLabel = "node-role.kubernetes.io/master"
+	backupPath      = "/etc/kubernetes/cluster-backup"
+)
+
+func TestBackupScript(t *testing.T) {
+	clientSet := framework.NewClientSet("")
+	masterNodes, err := clientSet.CoreV1Interface.Nodes().List(context.TODO(), metav1.ListOptions{LabelSelector: masterNodeLabel})
+	require.NoErrorf(t, err, "error while listing master nodes")
+
+	// create debug pod on first master node
+	debugNodeName := masterNodes.Items[0].Name
+	debugNodeAsArg := fmt.Sprintf("debug node/%s %s %s", debugNodeName, "--preserve-pod=true", "--as-root=true")
+	err = exec.Command("oc", debugNodeAsArg).Run()
+	require.NoError(t, err)
+
+	// see https://www.redhat.com/sysadmin/how-oc-debug-works
+	debugPodName := debugNodeName + "-debug"
+	debugPodName = strings.ReplaceAll(debugPodName, ".", "-")
+
+	// verify no backup exist
+	cmdAsStr := fmt.Sprintf("ls -l %s", backupPath)
+	output, err := exec.Command("oc", getOcArgs(debugPodName, cmdAsStr)).CombinedOutput()
+	require.NoError(t, err)
+	outputString := string(output)
+	require.Emptyf(t, outputString, "expected no backup, found %s", outputString)
+
+	cmdAsStr = "/host"
+	err = exec.Command("oc", getOcArgs(debugPodName, cmdAsStr)).Run()
+	require.NoError(t, err)
+
+	cmdAsStr = fmt.Sprintf("/usr/local/bin/cluster-backup.sh --force %s", backupPath)
+	err = exec.Command("oc", getOcArgs(debugPodName, cmdAsStr)).Run()
+	require.NoError(t, err)
+
+	// verify backup created
+	cmdAsStr = fmt.Sprintf("ls -l %s", backupPath)
```

Comment:
you need to use `find`, ls is not recursive. Checkout https://github.com/openshift/cluster-etcd-operator/blob/master/test/e2e/backup_test.go#L543